ui/src/components/MainContainer.js

```diff
@@ -228,6 +228,12 @@ const MainContainer = () => {
           permissions={['public']}
           component={AboutContainer}
         />
+        <PrivateRoute
+          exact
+          path="/with-everything"
+          permissions={['noUser']}
```

Comment:
you probably have to rebase again. latest retreat branch has this permission as public